styles/arch-wiki/catppuccin.user.css

```diff
@@ -2,7 +2,7 @@
 @name Arch Wiki Catppuccin
 @namespace github.com/catppuccin/userstyles/styles/arch-wiki
 @homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/arch-wiki
-@version 0.0.8
+@version 0.0.9
```

Comment:
should be a major version for a rewrite

Reply:
*minor tbf

Reply:
shouldn't it be major because this is a entire rewrite?